config/src/client.rs

```diff
@@ -97,31 +104,29 @@ impl Default for ConfigurationProxy {
             basic_auth: Some(None),
             torii_api_url: None,
             torii_telemetry_url: None,
-            transaction_time_to_live_ms: Some(DEFAULT_TRANSACTION_TIME_TO_LIVE_MS),
+            transaction_time_to_live_ms: Some(None),
```

Comment:
I would still argue that we need a default for this.

Reply:
check the comment [here](https://github.com/hyperledger/iroha/pull/3485/files#diff-38804a7b8ccf61b62b61b4f22618d2c5a9ec747a42041d54ac4cfd44239fd79eR92)

Reply:
can you give an argument for it?

Reply:
We have this option in general because blockchain operators want to fight DoS attacks and manage bandwidth. Given how much is stored in RAM, this value must be set as low as possible, and easy to change. 

IMO a low reasonable value does the job just as well as asking the user to set it explicitly. Remember this is the client configuration, so we're probably dealing with the front-end where the Swift or Java program can figure out the value on its own, which is likely to just pick a default. This way, this is a default that's controlled upstream.